seed/checkRoleExistence.js

```diff
@@ -0,0 +1,38 @@
+const Role = require('~/models/role')
+const {
+  roles: { STUDENT, MENTOR, ADMIN, SUPERADMIN }
+} = require('~/consts/auth')
+const SeedRole = require('~/seed/seedRole')
+const logger = require('~/logger/logger')
+
+const checkRoleExistence = async () => {
```

Comment:
In my opinion, this naming isn't really clear. If you were to just check for role existence, then sure. But you also create new role if they don't exist, hence you have side effects in this function. Maybe it'd be better to change the naming to something like `createRolesIfNotExist`

Reply:
I must disagree with your comment coz in fact all what it does is checking for the existence and then calling the function that creates this role. 
if I had logic that creates this role there then i would agree with your comment